### PR TITLE
EAD-Session-1

### PR DESCRIPTION
**Pull request by meta-manoj-sharma**

### CODE REVIEW

File: EAD-Session-1/EADsession1.html [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+<h1 id="main">Metacube Parking System</h1>
+<hr>
+<div align="center" class="divisionOne" id="firstForm">
+<form  name="employee" action="" onsubmit="checkForm()" >
+<h2>
+Employee Registration Form
+</h2>
+<br>
+<label ><b>Name</b></label>
+<input name="name" type="text" placeholder="Enter full name">
```
Comment (jbhavya) [inline EAD-Session-1/EADsession1.html]: Employee name should not accept any input other than valid characters.

File: EAD-Session-1/EADsession1.html [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+<label><b>Vehicle Number</b></label>
+<input type="text" placeholder="Enter Vehicle Number">
+<br><br>
+
+<label ><b>Employee ID</b></label>
+<input id="employee" type="text" readonly>
+<br><br>
+
+<label><b>Identification</b></label>
+<textarea rows="6">
```
Comment (jbhavya) [inline EAD-Session-1/EADsession1.html]: Cursor position should be on top left on vehicle reg form

File: EAD-Session-1/EADsession1.html [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+<div id="fourWheelerPassUSD" align="center">
+<label>
+<input type="radio" name="fourWheelerPass" value="0.28$">Daily - 0.28 $ <br><br>
+</label>
+
+<label>
+<input type="radio" name="fourWheelerPass" value="6.94$">Monthly - 6.94$ <br><br>
+</label>
+<label>
+<input type="radio" name="fourWheelerPass" value="48.60$">Yearly - 48.60 <br><br>
```
Comment (jbhavya) [inline EAD-Session-1/EADsession1.html]: $ symbol is missing.

File: EAD-Session-1/EADsession1.html [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+
+
+<label><b>Password</b></label>
+<input name="password" type="password" placeholder="Enter Password">
+<br><br>
+<label><b>Confirm Password</b></label>
+<input type="password" name="confirm" placeholder="Confirm Password">
+<br><br>
+<label><b>Contact Number</b></label>
+<input type="number" name="contact" placeholder="Enter Contact Number">
```
Comment (jbhavya) [inline EAD-Session-1/EADsession1.html]: what is the max length for contact no ?

File: EAD-Session-1/EADsession1.html (hunk 1 above)
Comment (meta-manoj-sharma) [reply EAD-Session-1/EADsession1.html]: logic updated

File: EAD-Session-1/EADsession1.html (hunk 2 above)
Comment (meta-manoj-sharma) [reply EAD-Session-1/EADsession1.html]: issue resolved

File: EAD-Session-1/EADsession1.html (hunk 3 above)
Comment (meta-manoj-sharma) [reply EAD-Session-1/EADsession1.html]: issue resolved

File: EAD-Session-1/EADsession1.html (hunk 4 above)
Comment (meta-manoj-sharma) [reply EAD-Session-1/EADsession1.html]: issue resolved

**Pull request merged**